server/src/server.ts

```diff
@@ -82,6 +70,8 @@ const completionProvider: YmlCompletionItemsProvider = new YmlCompletionItemsPro
 // supports full document sync only
 const documents: TextDocuments = new TextDocuments();
 
+let SETTINGS: IYseopmlServerSettings;
```

Comment:
> I think that even if it is not a `const` it feels better to have it with full caps since it is a global variable. I can rename it if required.

Not entirely convinced (the places where you assign values to it hurt my eyes xD) but not entirely against it.

Reply:
I will rename it, then. At least to use the same standards as for the other global constants (such as `documents`).